Makefile

```diff
@@ -44,6 +48,7 @@ ZK_JAR=$(ZK_JAR_PATH)/$(ZK_JAR_NAME)
 SHELL = /bin/bash
 
 prepareLic:
+	echo 'This makefile is for ci test and have dependency, do not run locally, if you want to run unit test, please run `go test ./...` directly'
```

Comment:
echo 'The makefile is for ci test and has dependencies. Do not run locally. If you want to run the unit tests, run command`go test ./...` directly'

Reply:
The suggestion from Mr.Yu is reasonable, and I'll fix it right away.

Reply:
Mr.Yu? haha.